### ISSUE TITLE
Error `npm start` on Windows

### ISSUE THREAD

**Issue by moonlitangel** [pjhjohn/jsonplaceholder-client#22]
Running `npm start` error :

```
[5] start:json-server' exited with code 1
[4] run exited with code 1
[3] np exited with code 1
[2] start:client' exited with code 1
[1] run exited with code 1
[0] np exited with code 1

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\thfql\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.6
npm ERR! code ELIFECYCLE
npm ERR! jsonplaceholder-client@0.0.1 start: `concurrently --kill-others 'npm run start:client' 'npm run start:json-server'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jsonplaceholder-client@0.0.1 start script 'concurrently --kill-others 'npm run start:client' 'npm run start:json-server''.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jsonplaceholder-client package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     concurrently --kill-others 'npm run start:client' 'npm run start:json-server'
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jsonplaceholder-client
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jsonplaceholder-client
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\thfql\Documents\react-first\jsonplaceholder-client\npm-debug.log
```

`npm-debug.log` is:

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\thfql\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@3.10.6
3 info using node@v6.3.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle jsonplaceholder-client@0.0.1~prestart: jsonplaceholder-client@0.0.1
6 silly lifecycle jsonplaceholder-client@0.0.1~prestart: no script for prestart, continuing
7 info lifecycle jsonplaceholder-client@0.0.1~start: jsonplaceholder-client@0.0.1
8 verbose lifecycle jsonplaceholder-client@0.0.1~start: unsafe-perm in lifecycle true
9 verbose lifecycle jsonplaceholder-client@0.0.1~start: PATH: C:\Users\thfql\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin;C:\Users\thfql\Documents\react-first\jsonplaceholder-client\node_modules\.bin;C:\Users\thfql\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\thfql\bin;C:\Program Files (x86)\Intel\iCLS Client;C:\Program Files\Intel\iCLS Client;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Git\cmd;C:\Program Files\nodejs;C:\Python27;C:\Python27\Scripts;C:\Program Files\Intel\WiFi\bin;C:\Program Files\Common Files\Intel\WirelessCommon;C:\Users\thfql\AppData\Local\Programs\Python\Python35\Scripts;C:\Users\thfql\AppData\Local\Programs\Python\Python35;C:\Users\thfql\AppData\Roaming\npm;C:\Users\thfql\AppData\Local\atom\bin;C:\Program Files\Intel\WiFi\bin;C:\Program Files\Common Files\Intel\WirelessCommon;%USERPROFILE%\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl
10 verbose lifecycle jsonplaceholder-client@0.0.1~start: CWD: C:\Users\thfql\Documents\react-first\jsonplaceholder-client
11 silly lifecycle jsonplaceholder-client@0.0.1~start: Args: [ '/d /s /c',
11 silly lifecycle   'concurrently --kill-others \'npm run start:client\' \'npm run start:json-server\'' ]
12 silly lifecycle jsonplaceholder-client@0.0.1~start: Returned: code: 1  signal: null
13 info lifecycle jsonplaceholder-client@0.0.1~start: Failed to exec start script
14 verbose stack Error: jsonplaceholder-client@0.0.1 start: `concurrently --kill-others 'npm run start:client' 'npm run start:json-server'`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (C:\Users\thfql\AppData\Roaming\npm\node_modules\npm\lib\utils\lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (C:\Users\thfql\AppData\Roaming\npm\node_modules\npm\lib\utils\spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:852:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
15 verbose pkgid jsonplaceholder-client@0.0.1
16 verbose cwd C:\Users\thfql\Documents\react-first\jsonplaceholder-client
17 error Windows_NT 10.0.14393
18 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\thfql\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
19 error node v6.3.0
20 error npm  v3.10.6
21 error code ELIFECYCLE
22 error jsonplaceholder-client@0.0.1 start: `concurrently --kill-others 'npm run start:client' 'npm run start:json-server'`
22 error Exit status 1
23 error Failed at the jsonplaceholder-client@0.0.1 start script 'concurrently --kill-others 'npm run start:client' 'npm run start:json-server''.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the jsonplaceholder-client package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     concurrently --kill-others 'npm run start:client' 'npm run start:json-server'
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs jsonplaceholder-client
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls jsonplaceholder-client
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

Direct run script `concurrently --kill-others 'npm run start:client' 'npm run start:json-server'` and `npm run start:client`, `npm run start:json-server` is OK. But `npm start` running error.

Mac device is successful , but Windows device is failed .

**Comment by MoonlitAngel** [comment]
package.json :
```
"start": "concurrently --kill-others \"npm run start:client\" \"npm run start:json-server\"",
```

**Comment by pjhjohn** [comment]
https://github.com/billyvg/pokemon-journal/issues/11 didn't fix the problem

**Comment by pjhjohn** [comment]
```
run exited with code 1
np exited with code 1
```

에서 np가 npm이 잘못 출력된 것 같음.
`np exited with code 1` 로 검색 -> https://github.com/jupyter/help/issues/47

**Comment by pjhjohn** [comment]
Fixed in cebc096

**Comment by pjhjohn** [comment]
Fixed in c681178
